src/main/java/com/abdullah/coding/challenge/controllers/CustomerController.java

```diff
@@ -0,0 +1,30 @@
+package com.abdullah.coding.challenge.controllers;
+
+import com.abdullah.coding.challenge.Response;
+import com.abdullah.coding.challenge.services.BookingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/customer")
+public class CustomerController {
+    @Autowired
+    private BookingService bookingService;
+
+    @PostMapping("/booking-comment")
```

Comment:
I believe that using PATCH would have made more sense because again I am only updating a specific property of a resource. I did not use request body because realistically, a comment can not be that long such that it would exceed the number of characters allowed in a request query, this however depends on the business logic. If the business logic stated that a customer could write huge comments, then sure I would go with request body.